tools/upmap/upmap-remapped.py

```diff
@@ -93,29 +112,44 @@ def rm_upmap_pg_items(pgid):
 
 # discover remapped pgs
 try:
-  remapped_json = subprocess.getoutput('ceph pg ls remapped -f json | jq -r')
-  remapped = json.loads(remapped_json)
+  if use_shell:
+    remapped_json = subprocess.getoutput('ceph pg ls remapped -f json | jq -r')
+  else:
+    cmd = {"prefix": "pg ls", "states": ["remapped"], "format": "json"}
+    ret, output, err = cluster.mon_command(json.dumps(cmd), b'', timeout=5)
+    remapped_json = output.decode('utf-8').strip()
+  try:
+    remapped = json.loads(remapped_json)['pg_stats']
+  except KeyError:
+    eprint("There are no remapped PGs")
+    sys.exit(0)
 except ValueError:
   eprint('Error loading remapped pgs')
   sys.exit(1)
 
-# nautilus compat
```

Comment:
Nautilus is 5 major versions ago. No need to mention when the change to the json output happened. Also moved it inside of the initial try block.